src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -273,6 +273,10 @@ public TimeSpan Expect100ContinueTimeout
             }
         }
 
+        public bool SupportsAutomaticDecompression => true;
+        public bool SupportsProxy => true;
+        public bool SupportsRedirectConfiguration => true;
```

Comment:
Should they be `internal` because this is a `public` type as mentioned in the commit message?

Reply:
Humm you are absolutely right.  It seems I committed the wrong working repo branch.

Reply:
Thanks for the catch.  The comments are addressed.